### DISCUSSION TITLE
It should be easier

### DISCUSSION THREAD

**Discussion by psociety** [Gurkengewuerz/nitro-docker#3] [General]
I don't understand why the fork is worse than the original repo :S

1. The original repo takes advantage of the fact that when setting up the docker db, the system will automatically run all .sql files placed on a specific folder. Here you removed the folder and then manually listed all sql files to download and run as well as some queries that could perfectly be an .sql file on said folder too :S

2. Same happens with:
```
wget https://github.com/billsonnn/nitro-react/files/10334858/room.nitro.zip && \
unzip -o room.nitro.zip -d assets/assets/bundled/generic && \
````

This will only work on linux, it could simply be executed inside the linux container using an entrypoint, so it works on all platforms :S

3. The main idea of docker is to have the app contained, yet this repo requires installing node on the root machine to run commands rather than using a container for it

**Comment by Gurkengewuerz** [comment]
hey 👋🏼
firstly this repository was developed while I was testing nitro. any PR's are very welcome, so if you have any enhancements, just create a PR.

1. arcturus have no automatic migrations implemented. This is why you have to keep it manually in order to preserve any errors for migrations!

2. all my repositories are *unix first, because everything i deploy is on linux servers 😉 Windows users have to do it manually or in WSL2. But keep in mind PR's are welcome.

3. yeah i know, but the software typically must support docker in the one or other way. also i don't have time to maintain a docker container to skip the building process. All of this was developed simply in a few days to take a look at the current development state. I already know a few people also using this in production. This is why I will close this as an "issue" and converted it to a discussion.

I'm open for any pull request, likewise to add contributes

**Reply by psociety** [reply]
1. It's not arcturus, but the container itself!
If you place sql files in the folder listed here, they will be executed automatically.
https://github.com/Gurkengewuerz/nitro-docker/blob/main/docker-compose.yaml#L55

That's why holo has the `sql`s in that folder:
https://github.com/Holo5/nitro-docker/blob/main/docker-compose.yaml#L57
https://github.com/Holo5/nitro-docker/tree/main/mysql/dumps
